docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: "3"
```

Comment:
What's the purpose of this docker-compose file?
Also, would you mind changing it to from .yaml to .yml (for consistency)?